src/Symfony/Component/Routing/Loader/Configurator/ImportConfigurator.php

```diff
@@ -40,8 +40,14 @@ public function __destruct()
      *
      * @return $this
      */
-    final public function prefix($prefix)
+    final public function prefix($prefix, string $namePrefix = '')
```

Comment:
IMO addng a name prefix should be a separate method again. Those are totally unrelated concepts and I can add a name prefix without having to add a path prefix. This is also possible in the annotation loader for example. So it not consistent to do do it in one method.

Reply:
agreed, done in #26284